scripts/singularity_cmd

```diff
@@ -72,17 +72,52 @@ pass_git_config "user.name" "ReproNim User"
 pass_git_config "user.email" "nobody@example.com"
 
 # Common arguments for the singularity run
-SARGS=( -e -c -B "$PWD" -H "$BHOME" --pwd "$PWD" "$@" )
+SARGS=( -e -B "$PWD" -H "$BHOME" --pwd "$PWD" "$@" )
 
+# Due to https://github.com/sylabs/singularity/issues/3949
+# which was fixed only in v3.3.0-rc.1-431-g40331a5b1
+# for now we need to avoid using `-c` in such cases and
+# just create those needed bind points manually and bind mount
+# /tmp manually (apparently -W is not in effect really without -c).
+# TODO: make it also check/depend on current version of singularity
+need_no_c=
+for d in "$PWD" "$updir"; do
+	if [ "${d##/tmp}" != "$d" ]; then
```

Comment:
Should the pattern here be `/tmp/` (trailing slash) so that your workaround isn't unnecessarily applied to someone's, say, "/tmpa"?

Reply:
good catch! will push a fix here and some other spot IIRC